System.IO.Abstractions.TestingHelpers.Tests/MockDirectoryTests.cs

```diff
@@ -788,6 +788,20 @@ public void MockDirectory_GetFiles_Returns_Files()
             Assert.AreEqual(testPath, entries.First());
         }
 
+        [Test]
+        public void MockDirectory_GetFiles_Returns_Files_WithRelativePath()
+        {
+            // arrange
+            var fileSystem = new MockFileSystem(new Dictionary<string, MockFileData>());
+
+            string directory = XFS.Path(@"C:\foo");
+            fileSystem.Directory.SetCurrentDirectory(directory);
+            fileSystem.AddFile($@"C:\test.txt", new MockFileData("Some ASCII text."));
+
+            Assert.AreEqual(fileSystem.Directory.GetFiles(@"C:\").Length, 1); // Assert with absolute path
+            Assert.AreEqual(fileSystem.Directory.GetFiles(@"..\").Length, 1); // Assert with relative path
```

Comment:
You need `XFS.Path` around the path so that the slashes are normalized.

```suggestion
            Assert.AreEqual(fileSystem.Directory.GetFiles(XFS.Path(@"..\")).Length, 1);
```

Reply:
Made the changes and committed. Thanks.